src/native/fs/fs_napi.cpp

```diff
@@ -1887,6 +1887,8 @@ struct DirWrap : public Napi::ObjectWrap<DirWrap>
             {
                 InstanceMethod("close", &DirWrap::close),
                 InstanceMethod("read", &DirWrap::read),
+                InstanceMethod("telldir", &DirWrap::tell),
+                InstanceMethod("seekdir", &DirWrap::seek),
```

Comment:
I would keep the DirWrap methods named the same to avoid having to do a "lookup" when trying to locate a function name in the code, so DirWrap::telldir/seekdir is better IMHO (even that it can be understood already that this is about a directory from the class...).